### PR TITLE
Added Play Music on Hover Functions

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Few points for this PR:

- Currently using **the audio link** as ID (because unique), if we can have a field as post-id (which will be unique), I'll add it.
- Test it on LAN for no lag, no sound compression added till now.
- Don't worry about ugly CSS we have now, will soon fix it with @anshumanv

### CODE REVIEW

**Comment by anshumanv** [comment]
What CSS needs to be fixed? @aashutoshrathi

**Comment by anshumanv** [comment]
> Test it on LAN for no lag, no sound compression added till now.

Sound compression won't be added.

**Comment by aashutoshrathi** [comment]
@Monal5031 said it is very bad, will soon make up a new look and we will design together. :wink:

**Comment by anshumanv** [comment]
@aashutoshrathi test audio on the category page, please.

**Comment by aashutoshrathi** [comment]
I checked it is working fine :+1: 
Should we add more of posts of a different category for better testing?

**Comment by anshumanv** [comment]
Yes, please do the needful. :wink:

File: static/js/musicOnHover.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function playIt(x) {
+  var audio = document.getElementById(x);
```
Comment (anshumanv) [inline static/js/musicOnHover.js:2]: Is there a way to do it by querySelector?

Comment (aashutoshrathi) [reply static/js/musicOnHover.js:2]: No :smirk: .

Comment (anshumanv) [reply static/js/musicOnHover.js:2]: I tried and couldn't do it, so thought you may find.

File: blogview/templates/all_posts.html [hunk 2]
```diff
@@ -30,13 +30,23 @@
                     </div>
                   </div>
                 </div>
+                <audio id = "{{ post.music_link }}" controls style="display: none;">
```
Comment (mohitkyadav) [inline blogview/templates/all_posts.html:33]: @aashutoshrathi You are creating one audio tag for each post. Don't you think it's more data consuming.

**Review by mohitkyadav — COMMENTED** [review]
Why not create a single `audio` and change the `src ` on `mouseenter`?

File: blogview/templates/all_posts.html (hunk 2 above)
Comment (Monal5031) [reply blogview/templates/all_posts.html:33]: Any other suggestions then @mohitkyadav ?

**Review by Monal5031 — CHANGES_REQUESTED** [review]
Dont push db

File: blogview/templates/all_posts.html (hunk 2 above)
Comment (mohitkyadav) [reply blogview/templates/all_posts.html:33]: > Why not create a single audio and change the src on mouseenter?

Comment (anshumanv) [reply blogview/templates/all_posts.html:33]: Let's create audio in base.

Comment (aashutoshrathi) [reply blogview/templates/all_posts.html:33]: I think it is for the best, we already had an audio tag in each post. :confused:

**Comment by aashutoshrathi** [comment]
@mohitkyadav getting [this](https://developers.google.com/web/updates/2017/06/play-request-was-interrupted) when switched to one audio element on a page, also when I solved it, it only plays the same song on all posts in a page. 
Should I add these things to other PR?

**Comment by aashutoshrathi** [comment]
@Monal5031 
I don't see what problem pushing DB will cause as of now.
Yes, I know its a bad practice in the long run but here Testing becomes easy due to preset posts.
cc: @anshumanv @mohitkyadav

**Comment by anshumanv** [comment]
Agreed with @aashutoshrathi we can try to be more professional once we're in production.

**Review by mohitkyadav — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]
LGTM

**Comment by anshumanv** [comment]
Merging this as WIP.

**Pull request merged**